build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:$build_tools_version"
+        classpath 'com.android.tools.build:gradle:2.3.2'
```

Comment:
you should change the value of build_tools_version instead of setting it directly (it is also used for the databinding version).